src/main/java/org/opentripplanner/api/resource/GraphInspectorVectorTileResource.java

```diff
@@ -0,0 +1,125 @@
+package org.opentripplanner.api.resource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.function.Predicate;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+import org.glassfish.grizzly.http.server.Request;
+import org.opentripplanner.api.model.TileJson;
+import org.opentripplanner.inspector.vector.AreaStopsLayerBuilder;
+import org.opentripplanner.inspector.vector.LayerBuilder;
+import org.opentripplanner.inspector.vector.LayerParameters;
+import org.opentripplanner.inspector.vector.VectorTileUtils;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.standalone.api.OtpServerRequestContext;
+import org.opentripplanner.transit.service.TransitService;
+
+/**
+ * Slippy map vector tile API for rendering various graph information for inspection/debugging
+ * purposes.
+ */
+@Path("/routers/{ignoreRouterId}/inspector/vectortile")
+public class GraphInspectorVectorTileResource {
+
+  public static final String APPLICATION_X_PROTOBUF = "application/x-protobuf";
+
+  private static final List<LayerParameters<LayerType>> DEBUG_LAYERS = List.of(
+    new LayerParams("areaStops", LayerType.AreaStop)
+  );
+
+  private final OtpServerRequestContext serverContext;
+  private final String ignoreRouterId;
+
+  public GraphInspectorVectorTileResource(
+    @Context OtpServerRequestContext serverContext,
+    /**
+     * @deprecated The support for multiple routers are removed from OTP2.
+     * See https://github.com/opentripplanner/OpenTripPlanner/issues/2760
+     */
+    @Deprecated @PathParam("ignoreRouterId") String ignoreRouterId
```

Comment:
Now that we have the opportunity, do we want maybe not introduce this or add it and then remove it in one go for all of the resources?